dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/icon/IconControllerTest.java

```diff
@@ -60,88 +64,134 @@ void shouldCreateCustomIconWhenFileResourceExist() throws IOException {
     assertEquals(fileResourceId, response.getString("fileResourceUid").string());
     assertEquals(keywords, response.getArray("keywords").toString());
     assertEquals(getCurrentUser().getUid(), response.getString("userUid").string());
     assertEquals(
-        String.format(contextService.getApiPath() + "/icons/%s/icon", iconKey),
+        String.format(contextService.getApiPath() + "/icons/%s/icon", iconKey1),
         response.getString("href").string());
   }
 
   @Test
   void shouldUpdateIconWhenKeyExists() throws IOException {
     String updatedDescription = "updatedDescription";
     String updatedKeywords = "['new k1', 'new k2']";
-    createCustomIcon(createFileResource(), keywords);
+    createCustomIcon(createFileResource(), keywords, iconKey1);
 
     JsonObject response =
         PUT(
                 "/icons",
                 "{'key':'"
-                    + iconKey
+                    + iconKey1
                     + "', 'description':'"
                     + updatedDescription
                     + "', 'keywords':"
                     + updatedKeywords
                     + "}")
             .content();
 
-    assertEquals(String.format("Icon %s updated", iconKey), response.getString("message").string());
+    assertEquals(
+        String.format("Icon %s updated", iconKey1), response.getString("message").string());
   }
 
   @Test
   void shouldGetOnlyCustomIcons() throws IOException {
     String fileResourceId = createFileResource();
-    createCustomIcon(fileResourceId, keywords);
+    createCustomIcon(fileResourceId, keywords, iconKey1);
+
+    JsonObject content = GET("/icons?keywords=k1").content(HttpStatus.OK);
 
-    JsonList<JsonIcon> icons =
-        GET("/icons?keywords=k1").content(HttpStatus.OK).asList(JsonIcon.class);
+    JsonList<JsonIcon> icons = content.getList("icons", JsonIcon.class);
 
     assertCustomIcons(icons.get(0), keywords, fileResourceId);
   }
 
   @Test
   void shouldGetIconsWithSpecifiedKey() throws IOException {
     String fileResourceId = createFileResource();
-    createCustomIcon(fileResourceId, keyword);
+    createCustomIcon(fileResourceId, keyword, iconKey1);
+
+    JsonObject content = GET("/icons?keywords=m1").content(HttpStatus.OK);
 
-    JsonList<JsonIcon> icons =
-        GET("/icons?keywords=m1").content(HttpStatus.OK).asList(JsonIcon.class);
+    JsonList<JsonIcon> icons = content.getList("icons", JsonIcon.class);
 
     assertCustomIcons(icons.get(0), keyword, fileResourceId);
   }
 
   @Test
   void shouldDeleteIconWhenKeyExists() throws IOException {
-    createCustomIcon(createFileResource(), keywords);
+    createCustomIcon(createFileResource(), keywords, iconKey1);
+
+    JsonObject response = DELETE(String.format("/icons/%s", iconKey1)).content();
+
+    assertEquals(
+        String.format("Icon %s deleted", iconKey1), response.getString("message").string());
+  }
+
+  @Test
+  void shouldGetIconsWithPager() throws IOException {
+
```

Comment:
Please remove these empty lines at the start of methods. Maybe there is an IDE setting 🤷🏻